DOCKER/docker-entrypoint.sh

```diff
@@ -9,7 +9,7 @@ if [ ! -d "$TMHOME/config" ]; then
 		-e "s/^proxy_app\s*=.*/proxy_app = \"$PROXY_APP\"/" \
 		-e "s/^moniker\s*=.*/moniker = \"$MONIKER\"/" \
 		-e 's/^addr_book_strict\s*=.*/addr_book_strict = false/' \
-		-e 's/^timeout_commit\s*=.*/timeout_commit = "500ms"/' \
+		-e 's/^target_round_duration\s*=.*/target_round_duration = "1000ms"/' \
```

Comment:
`target_round_duration` seems like the previous name for `target_height_duration`

```suggestion
		-e 's/^target_height_duration\s*=.*/target_height_duration = "1000ms"/' \
```

Reply:
https://github.com/celestiaorg/celestia-core/pull/965/commits/8e5b17b05ac3f4c03d9ae7c9eed0155cd49ebaff